cmd/bootkube/render.go

```diff
@@ -113,6 +117,12 @@ func validateRenderOpts(cmd *cobra.Command, args []string) error {
 	if renderOpts.apiServers == "" {
 		return errors.New("Missing requried flag: --api-servers")
 	}
+	if renderOpts.networkProvider != asset.NetworkFlannel && renderOpts.networkProvider != asset.NetworkCalico {
+		return errors.New("Must specify --network-provider flannel or calico")
+	}
+	if renderOpts.networkProvider == asset.NetworkCalico && renderOpts.calicoNetworkPolicy {
+		return errors.New("Cannot specify --experimental-calico-network-policy with --network-provider=calico, Calico supports network policy automatically.")
```

Comment:
Maybe we just drop this altogether and put this under network provider instead?

`--network-provider={flannel,experimental-calico-network-policy, experimiental-calico-bgp}`?

I'm leaning toward the experimental because if we don't have tests - we could break it at any time.

Reply:
Experimental prefixes work for me. I prefer we say "calico" rather than "calico-bgp", since bgp is redundant. I'd also prefer to keep "canal-style" policy-only projects separate from CNI providers, long term it doesn't really belong here since its not (supposed to be) providing networking or really essential to bringing up a minimal control plane. Suppose some other network policy app comes along. Its an addon.

Reply:
`--network-provider={flannel,experimental-calico}`